.env.example

```diff
@@ -1,4 +1,3 @@
-DAPI_CLIENT_SEEDS=ip1,ip2
-DAPI_CLIENT_PORT=3000
+DAPI_SEED= # Format - ip:port
```

Comment:
This comment will be a part of value. You need to start a line with sharp in order to comment.